### PR TITLE
(FACT-1967) document cmake flags

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,14 +91,36 @@ for (auto const& result : results) {
...
 
-```
+```sh
 mkdir build-release
 cd build-release
 cmake ..
 ```
 
-Optionally, also prepare the debug environment:
+Starting with GCC 8.0, some warnings will arise, which will run into an error.
```
Comment (GabrielNagy) [inline README.md]: We shouldn't need these since https://github.com/puppetlabs/libwhereami/pull/43 will take care of the warnings.

Comment (bastelfreak) [reply README.md]: fixed it

**Review by GabrielNagy — APPROVED** [review]
:+1:

**Comment by GabrielNagy** [comment]
can you squash this commit please https://github.com/puppetlabs/libwhereami/pull/44/commits/ec69079d3db8b80da424428efe6c285ddae4f36e

thanks!

**Comment by GabrielNagy** [comment]
@bastelfreak ping

**Comment by bastelfreak** [comment]
@GabrielNagy done

**Pull request merged**